create-argo/values.yaml

```diff
@@ -1,419 +1,271 @@
+      {{- if or .Values.postgresql.enabled (and .Values.externalDatabase.enabled (eq .Values.externalDatabase.type "postgresql")) }}
+      postgresql:
+      {{- else if or .Values.mysql.enabled (and .Values.externalDatabase.enabled (eq .Values.externalDatabase.type "mysql")) }}
+      mysql:
+      {{- end }}
+        host: {{ include "argo-workflows.controller.database.host" . }}
+        port: {{ include "argo-workflows.controller.database.port" . }}
+        database: {{ include "argo-workflows.controller.database" . }}
+        tableName: argo_workflows
+        userNameSecret:
+          name: {{ include "argo-workflows.controller.database.username.secret" . }}
+          key: username
+        passwordSecret:
+          name: {{ include "argo-workflows.controller.database.password.secret" . }}
+          key: {{ include "argo-workflows.controller.database.password.secret.key" . }}
+    {{- end }}
+    {{- if .Values.controller.workflowDefaults }}
+    workflowDefaults: {{- include "common.tplvalues.render" (dict "value" .Values.controller.workflowDefaults "context" $) | nindent 2 }}
+    {{- end }}
+    {{- if and .Values.server.auth.enabled .Values.server.auth.sso.enabled }}
+    sso: {{- include "common.tplvalues.render" (dict "value" .Values.server.auth.sso.config "context" $) | nindent 2 }}
+    {{- end }}
   instanceID:
     enabled: false
     useReleaseName: false
-
     explicitID: ""
-
-  logging:
-    level: info
-    globallevel: "0"
-    format: "text"
-
-  serviceType: ClusterIP
-  serviceAnnotations: {}
-  serviceLabels: {}
-  loadBalancerSourceRanges: []
-
-  resources:
-    requests:
-      memory: "256Mi"
-      cpu: "100m"
-    limits:
-      memory: "512Mi"
-      cpu: "1"
-
-  livenessProbe:
-    httpGet:
-      port: 6060
-      path: /healthz
-    failureThreshold: 3
-    initialDelaySeconds: 90
-    periodSeconds: 60
-    timeoutSeconds: 30
-
-  extraEnv:
-  - name: DEFAULT_REQUEUE_TIME
-    value: ${REQUEUE_TIME}
-  extraArgs:
-  - "--managed-namespace"
-  - "${NAMESPACE}"
-  volumeMounts: []
-  volumes: []
-  replicas: 1
-  revisionHistoryLimit: 10
-
-  pdb:
-    enabled: false
-
-  nodeSelector:
-    "cosmotech.com/tier": "services"
-  tolerations:
-  - key: "vendor"
-    operator: "Equal"
-    value: "cosmotech"
-    effect: "NoSchedule"
-  affinity: {}
-
-  topologySpreadConstraints: []
-
-  priorityClassName: ""
-
-  links: []
-  columns: []
-  navColor: ""
   clusterWorkflowTemplates:
     enabled: false
-  extraContainers: []
-
-  extraInitContainers: []
-
-  retentionPolicy: {}
-
-  nodeEvents:
+  metrics:
     enabled: true
-
-  kubeConfig: {}
-
-  podGCGracePeriodSeconds:
-
-  podGCDeleteDelayDuration: ""
-
-mainContainer:
-  imagePullPolicy: IfNotPresent
+    path: /metrics
+    serviceMonitor:
+      enabled: true
+  telemetry:
+    enabled: false
+    path: /telemetry
+  workflowWorkers: 32
+  workflowNamespaces:
+    - default
+  workflowDefaults: {}
```

Comment:
The `podGC` (and others values) should be configured here, we lost that from the previous config.

Reply:
Updated